### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getCursor' of undefined

### ISSUE THREAD

**Issue by adlermedrado** [Arcath/jekyll-atom#34]
Tried to open Data File and got an error:

**Atom Version**: 1.0.3
**System**: Mac OS X 10.10.4
**Thrown From**: [jekyll](https://github.com/arcath/jekyll-atom) package, v1.0.0

### Stack Trace

Uncaught TypeError: Cannot read property 'getCursor' of undefined

```
At /Users/adler/.atom/packages/jekyll/lib/jekyll.coffee:107

TypeError: Cannot read property 'getCursor' of undefined
  at Object.module.exports.openData (/Users/adler/.atom/packages/jekyll/lib/jekyll.coffee:107:24)
  at atom-workspace.<anonymous> (/Users/adler/.atom/packages/jekyll/lib/jekyll.coffee:56:65)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:241:29)
  at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61)
  at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:156:19)
  at EventEmitter.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:70:30)
  at emitOne (events.js:77:13)
  at EventEmitter.emit (events.js:166:7)

```

### Commands

```
     -6:36 core:copy (atom-text-editor.editor.is-focused)
     -6:35.6.0 editor:move-to-first-character-of-line (atom-text-editor.editor.is-focused)
  9x -6:35.1.0 core:select-right (atom-text-editor.editor.is-focused)
  2x -6:34.4.0 core:select-left (atom-text-editor.editor.is-focused)
     -6:33.4.0 core:backspace (atom-text-editor.editor.is-focused)
 21x -6:33 core:move-right (atom-text-editor.editor.is-focused)
     -6:31 core:move-left (atom-text-editor.editor.is-focused)
     -6:30.7.0 editor:select-to-end-of-line (atom-text-editor.editor.is-focused)
  2x -6:30.3.0 core:backspace (atom-text-editor.editor.is-focused)
  2x -6:29.6.0 editor:newline (atom-text-editor.editor.is-focused)
     -6:28.8.0 core:paste (atom-text-editor.editor.is-focused)
     -6:28.5.0 editor:newline (atom-text-editor.editor.is-focused)
     -6:27.6.0 core:save (atom-text-editor.editor.is-focused)
  6x -0:42.5.0 core:close (atom-text-editor.editor.is-focused)
     -0:35.2.0 jekyll:toolbar (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-atom-dark-syntax.theme-northem-dark-atom-ui)
     -0:01.6.0 jekyll:open-data (button#toggleButton.btn)
```

### Config

```json
{
  "core": {
    "themes": [
      "northem-dark-atom-ui",
      "atom-dark-syntax"
    ]
  },
  "jekyll": {}
}
```

### Installed Packages

```coffee
# User
Sublime-Style-Column-Selection, v1.3.0
atom-beautify, v0.28.8
autocomplete-ctags, v0.3.1
autocomplete-php, v0.3.6
ctags-status, v1.3.3
git-diff-details, v0.19.0
git-plus, v5.2.2
jekyll, v1.0.0
linter, v1.2.4
linter-php, v1.0.2
northem-dark-atom-ui, v0.2.1

# Dev
No dev packages
```

**Comment by Arcath** [comment]
Has this been fixed by the same fix as #35

**Comment by adlermedrado** [comment]
Great. 
I'll close this issue.
Thanks.
